projects/robots/micromagic/mantis/worlds/mantis.wbt

```diff
@@ -38,6 +38,6 @@ UnevenTerrain {
   perlinNOctaves 8
 }
 Mantis {
-  translation -24.12 61.42 -6.15
-  rotation 0 1 0 1.3
+  translation 6.1499 24.1193 61.4203
```

Comment:
```suggestion
  translation 6.15 24.12 61.42
```

Reply:
As they are just swapped around, I'd keep the same precision to avoid funny business.